<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #696969;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="side_frame">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid white;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="profile_frame">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>14</y>
         <width>212</width>
         <height>329</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="profile_button">
        <property name="geometry">
         <rect>
          <x>63</x>
          <y>40</y>
          <width>90</width>
          <height>90</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
border-radius: 50%;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>profile.png</normaloff>profile.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="name_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
color: white;
font-weight: bold;
</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="staff_id_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
color: white;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Staff ID</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="department_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
color: white;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Department</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="name_db">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
color: white;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="id_db">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
color: white;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="department_db">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
color: white;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="my_training_button">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>450</y>
         <width>211</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;
font-weight: bold;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>My Training</string>
       </property>
      </widget>
      <widget class="QPushButton" name="list_button">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>350</y>
         <width>211</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;
font-weight: bold;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>Training List</string>
       </property>
      </widget>
      <widget class="QPushButton" name="notification_button">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>550</y>
         <width>211</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;
font-weight: bold;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>Notification</string>
       </property>
      </widget>
      <widget class="QPushButton" name="logout_button">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>650</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
border-radius: 50%;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>logout.png</normaloff>logout.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
